lib/msal-angular/src/IMsalService.ts

```diff
@@ -15,6 +15,7 @@ import {
 import { Observable } from "rxjs";
 
 export interface IMsalService {
+    initialize(): Observable<void>;
```

Comment:
@jo-arroyo @jasonnutter Where's a good place to call this for angular?

Reply:
In the MsalRedirectComponent, I think.

Reply:
Does this mean, going forward, that any features requiring the `initialize` function need to include this component? Should we consider adding a new one that's not specific to redirects?

Reply:
Yes or invoking it manually, as is the case with redirects today. I'd rather have one bootstrap component than multiple.

Reply:
Done